htrc/torchlite/app.py

```diff
@@ -11,19 +14,29 @@
 from htrc.torchlite.filters import torchlite_stopword_filter, torchlite_stemmer, torchlite_lemmatizer
 from htrc.torchlite.middleware import TorchliteVersionHeaderMiddleware
 
-with open("config.yaml", mode="r", encoding="utf-8") as f:
-    config = safe_load(f)
+config_file: Optional[str] = os.getenv("TORCHLITE_CONFIG")
 
+if config_file is None:
+    raise OSError("Torchlite Configuration not found")
 
-def set_defaults(app: Torchlite, ef_api: Api) -> None:
-    if "featured_worksets_url" in config:
-        url = config["featured_worksets_url"]
-        if url:
-            r = requests.get(url)
-            if r.status_code == 200:
-                sample_workset_data = load(r.text, Loader=Loader)
-                for data in sample_workset_data:
-                    app.add_workset(**data)
+if config_file.startswith("http"):
```

Comment:
I thought we'd also support valid file urls...like `file:///blab/config.yaml`

Reply:
Since the code is working, I'm going to merge this pr but make a ticket: https://github.com/htrc/torchlite-backend/issues/39